test/pykernel/gen/colman_fused_eltwise_bcast.py

```diff
@@ -45,13 +48,28 @@ async def compute():
         
         for ct in range(start_col_tile, end_col_tile):
             # Reuse C across rows of A, B
-            c_block = c_in_cb.wait().pop()
+            # The following waits until *shape* number of tiles (1 in this case) is available and returns a RingView object as defined in:
+            # https://github.com/tenstorrent/tt-lang/blob/main/python/sim/cbsim/ringview.py.
+            # TODO: Perhaps consider making RingView pointers that come from wait()/reserve() read/write only respectively?
+            c_block = c_in_cb.wait() 
```

Comment:
Does the Python interface allow the number of tiles to be specified (with a default of 1)?